### PR TITLE
test: please disregard

### PR DESCRIPTION
**Pull request by jlebon**

### CODE REVIEW

File: README-design.md [hunk 1]
```diff
@@ -34,3 +34,4 @@ running as a Kube Job, and then trigger a container build which aggregates it
 
 The second model is to do a direct container build - that's what the `Dockerfile`
 in this repository is doing today.
+
```
Comment (ashcrow) [inline README-design.md]: Disregarding

Comment (cgwalters) [reply README-design.md]: I'm not actually replying

**Comment by smarterclayton** [comment]
Good, it caught a failure!

**Comment by smarterclayton** [comment]
Job should be on for all PRs and will be rerun before merge if anything else has been merged to master since it was last tested.

**Comment by jlebon** [comment]
Thanks for fixing!

**Pull request closed without merging**